src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The excavator started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The excavator stopped working.");

```

Comment:
Similarly, the stop message should clearly indicate that it is the Excavator that stopped working.